nrf_security/Kconfig

```
@@ -1459,6 +1459,14 @@ config MBEDTLS_SSL_CIPHERSUITES
 	  Warning: This field has offers no validation checks.
 	  MBEDTLS_SSL_CIPHERSUITES setting in mbed TLS config file.
 
+config NRF_SECURITY_DEFAULT_ENABLE_NONE
+	bool
+	help
+	  Disable all nRF Security features by default.
+	  It can save memory by disabling features that are not needed.
+	  This is promptless by purpose and state that another kconfig is the only way
+	  to set this configuration. The effect of the choice only works once.
+
```

```suggestion
	  Disable all nRF Security features.
	  It can save memory by disabling features that are not needed.
	  When enabled, if the project requires specific nRF Security features,
	  these features must be selected explicitly one by one.
	  This option is promptless and can be only set manually from within
	  a configuration file. The effect of disabling nRF Security features
	  only works once.
```
Added some lines discussed with Eduardo. Still, I don't understand the last sentence, and it needs to be clarified.

---

Description has been updated, I hope it's ok now.